rules/unicorn.js

```diff
@@ -1,102 +1,49 @@
     'unicorn/consistent-destructuring': 'error',
-    'unicorn/consistent-function-scoping': 'error',
     'unicorn/custom-error-definition': 'error',
     // Code should be formatted using Prettier.
     'unicorn/empty-brace-spaces': 'off',
-    'unicorn/error-message': 'error',
-    'unicorn/escape-case': 'error',
     'unicorn/expiring-todo-comments': [
       'error',
       { allowWarningComments: false, ignoreDatesOnPullRequests: false },
     ],
-    'unicorn/explicit-length-check': ['error', { 'non-zero': 'greater-than' }],
     'unicorn/filename-case': ['error', { case: 'camelCase' }],
     // No good use cases yet.
     'unicorn/import-style': 'off',
-    'unicorn/new-for-builtins': 'error',
     // Covered by eslint-comments/no-unlimited-disable.
     'unicorn/no-abusive-eslint-disable': 'off',
     // Unnecessary with TypeScript.
     'unicorn/no-array-callback-reference': 'off',
-    'unicorn/no-array-for-each': 'error',
     // Covered by functional/no-this-expressions.
     'unicorn/no-array-method-this-argument': 'off',
-    'unicorn/no-array-push-push': 'error',
     'unicorn/no-array-reduce': 'off',
-    'unicorn/no-console-spaces': 'error',
-    'unicorn/no-for-loop': 'error',
-    'unicorn/no-hex-escape': 'error',
-    'unicorn/no-instanceof-builtins': 'error',
-    'unicorn/no-keyword-prefix': 'off',
-    'unicorn/no-lonely-if': 'error',
     // Ternary operators help write immutable code.
     'unicorn/no-nested-ternary': 'off',
-    'unicorn/no-new-array': 'error',
     // This is covered by @typescript-eslint/no-deprecated
     'unicorn/no-new-buffer': 'off',
     'unicorn/no-null': 'off',
     // Unnecessary with TypeScript.
     'unicorn/no-object-as-default-parameter': 'off',
-    'unicorn/no-process-exit': 'off',
-    // Do not use classes in the first place.
-    'unicorn/no-static-only-class': 'off',
     // Covered by consistent-this.
     'unicorn/no-this-assignment': 'off',
+    // Covered by @typescript-eslint/return-await
+    'unicorn/no-unnecessary-await': 'off',
     'unicorn/no-unreadable-array-destructuring': 'off',
-    'unicorn/no-unused-properties': 'off',
-    'unicorn/no-useless-length-check': 'error',
-    'unicorn/no-useless-spread': 'error',
     'unicorn/no-useless-undefined': 'off',
-    'unicorn/no-zero-fractions': 'error',
     // Code should be formatted using Prettier.
     'unicorn/number-literal-case': 'off',
-    'unicorn/numeric-separators-style': 'error',
-    'unicorn/prefer-add-event-listener': 'error',
-    'unicorn/prefer-array-find': 'error',
-    'unicorn/prefer-array-flat-map': 'error',
-    'unicorn/prefer-array-index-of': 'error',
-    'unicorn/prefer-array-some': 'error',
-    // Not available in Node.js nor TypeScript yet.
-    'unicorn/prefer-at': 'off',
-    'unicorn/prefer-date-now': 'error',
-    'unicorn/prefer-default-parameters': 'error',
-    // Manual DOM manipulation is discouraged.
-    'unicorn/prefer-dom-node-append': 'off',
```

Comment:
not sure why turn these off, like if we do manipulate the DOM for some reason, why not have these suggestions